### PR TITLE
LPD-28657 Classic utility pages and navigation

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**
https://liferay.atlassian.net/browse/LPD-25423

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/layout-page-templates/master-pages/liferay-commerce-master/page-definition.json [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+					{
+						"definition": {
+							"htmlProperties": {
+								"htmlTag": "Footer"
+							},
+							"indexed": true,
+							"layout": {
+							}
+						},
+						"id": "5844652d-be2f-1117-5393-227a8ebf1cc5",
```
Comment (riccardo-alberti) [inline modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/layout-page-templates/master-pages/liferay-commerce-master/page-definition.json:143]: What is this for? is it correct to be in this format?

File: modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/layout-page-templates/master-pages/liferay-commerce-master/page-definition.json [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+													}
+												},
+												"fragmentLink": {
+												}
+											}
+										}
+									],
+									"indexed": true
+								},
+								"id": "ed49f66d-e226-5620-88bb-515594ec6f4f",
```
Comment (riccardo-alberti) [inline modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/layout-page-templates/master-pages/liferay-commerce-master/page-definition.json:338]: same

File: modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/layout-page-templates/master-pages/liferay-commerce-master/page-definition.json (hunk 1 above)
Comment (gianmarcobrunialti) [reply modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/layout-page-templates/master-pages/liferay-commerce-master/page-definition.json:143]: Exports left-overs, I can remove them, though they are ineffective.

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward:force

**Pull request closed without merging**